src/JBrowse/View/FastaFileDialog.js

```diff
@@ -81,7 +79,6 @@ return declare( FileDialog, {
                                     this.refSeqOrderChoice[1].checked ? "length descending" :
                                     undefined
                     })
-                    .then( () => this.dialog.hide(), err => { console.error(err) } )
```

Comment:
This removal causes the 'open sequence file' dialog in the web app to not close after you hit 'open'.  Can you fix that?

Reply:
This is the only remaining quibble I have with this pull req.